src/components/containers/section-container.js

```diff
@@ -1,15 +1,34 @@
-import styled from 'styled-components'
-import Box from './box'
+import styled, { css } from 'styled-components'
+import { Paddings } from '../../themes/function'
+import Box, { generateResponsiveStyles } from './box'
+import device from 'themes/device'
+
+const baseStyles = ({ p, pt, pl, pr, pb }) => css`
+    ${Paddings({ p, pt, pl, pr, pb })}
+`
+
+const responsiveStyles = generateResponsiveStyles(baseStyles)
 
 const SectionContainer = styled(Box).attrs({
     as: 'section',
 })`
     width: 100%;
-    padding: ${(props) => props.padding || '8rem 0'};
+    padding: ${(props) => props.padding || props.p || '8rem 0'};
     position: ${(props) => props.position || 'static'};
 
     /* prettier-ignore */
     background-color: var(--color-${(props) => props.background || 'white'});
+
+    @media ${device.tablet} {
+        padding: ${(props) => props.p || '8rem 0'};
+    }
+    @media ${device.tabletL} {
+        height: ${(props) => props.height};
+        padding: ${(props) => props.p};
+        padding-top: ${(props) => props.pt};
+    }
+
+    ${responsiveStyles}
```

Comment:
I think we don't need to add `responsiveStyles` here, It is already inherited in the parent component `Box`. The `props.p` condition alone can suffice the requirement.